flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/sink/writer/AsyncSinkWriter.java

```diff
@@ -307,6 +313,14 @@ public AsyncSinkWriter(
         initializeState(states);
     }
 
+    public AsyncSinkWriter(
```

Comment:
The `AsyncSinkWriter` needs to follow the changes of FLIP-372, so if we remove the deprecated methods, then AsyncSinkWriter could still work